src/theme/themes.ts

```diff
@@ -126,6 +133,13 @@ export const light: SaleorThemeColors = {
     mid: "#FEADAD",
     light: "#FEDEDE",
   },
+  errorAction: {
+    1: "#B63755",
+    2: "#D36474",
+    3: "#E9878B",
+    4: "#F7B6B2",
+    5: "#FBDDD8",
+  },
```

Comment:
If some colors are the same for dark and light, maybe it is worth to consider creating `const common: SaleorThemeColors` and extending/overwriting them in `export const dark: SaleorThemeColors` and `export const light: SaleorThemeColors` with what is needed.